File: Programming Basics/C++/Conditional Statements Advanced/SummerOutfit/SummerOutfit.cpp
#include <iostream>
using namespace std;

int main()
{
	int degrees;
	string partOfDay;
	cin >> degrees >> partOfDay;

	string outfit, shoes;

	if (degrees <= 18) {
		if (partOfDay == "Morning") {
			outfit = "Sweatshirt";
			shoes = "Sneakers";
		}
		else if (partOfDay == "Afternoon" || partOfDay == "Evening") {
			outfit = "Shirt";
			shoes = "Moccasins";
		}
	}
	else if (degrees <= 24) {
		if (partOfDay == "Morning" || partOfDay == "Evening") {
			outfit = "Shirt";
			shoes = "Moccasins";
		}
		else if (partOfDay == "Afternoon") {
			outfit = "T-Shirt";
			shoes = "Sandals";
		}
	}
	else {
		if (partOfDay == "Morning") {
			outfit = "T-Shirt";
			shoes = "Sandals";
		}
		else if (partOfDay == "Afternoon") {
			outfit = "Swim Suit";
			shoes = "Barefoot";
		}
		if (partOfDay == "Evening") {
			outfit = "Shirt";
			shoes = "Moccasins";
		}
	}

	cout << "It's " << degrees << " degrees, get your " << outfit << " and " << shoes << "." << endl;
}

File: Programming Basics/JavaScript/Conditional Statements Advanced/Exercises/summer-outfit.js
function solve(input) {
    const degrees = Number(input[0]);
    const partOfDay = input[1];

    let outfit = "",
        shoes = "";
    if (degrees <= 18) {
        if (partOfDay === "Morning") {
            outfit = "Sweatshirt";
            shoes = "Sneakers";
        } else if (partOfDay === "Afternoon" || partOfDay === "Evening") {
            outfit = "Shirt";
            shoes = "Moccasins";
        }
    } else if (degrees <= 24) {
        if (partOfDay === "Morning" || partOfDay === "Evening") {
            outfit = "Shirt";
            shoes = "Moccasins";
        } else if (partOfDay === "Afternoon") {
            outfit = "T-Shirt";
            shoes = "Sandals";
        }
    } else {
        if (partOfDay === "Morning") {
            outfit = "T-Shirt";
            shoes = "Sandals";
        } else if (partOfDay === "Afternoon") {
            outfit = "Swim Suit";
            shoes = "Barefoot";
        } else if (partOfDay === "Evening") {
            outfit = "Shirt";
            shoes = "Moccasins";
        }
    }

    console.log(`It's ${degrees} degrees, get your ${outfit} and ${shoes}.`);
}
